src/database/schema/schema.ts

```diff
@@ -0,0 +1,8 @@
+import { pgTable, uuid, text, timestamp, jsonb } from 'drizzle-orm/pg-core';
+
+export const planContent = pgTable('plan_content', {
+  id: uuid('id').primaryKey().defaultRandom(),
+  content: text('content').notNull(),
+  lastModified: timestamp('last_modified').notNull().defaultNow(),
+  vectorClock: jsonb('vector_clock').notNull(),
```

Comment:
What is this for? 🤔

Reply:
Im creating a schema table to wrap with drizzle. You can use ORM to query while you can't using raw query.